kin-sdk-core/src/main/java/kin/sdk/core/EncryptorImplV18.java

```diff
@@ -0,0 +1,129 @@
+package kin.sdk.core;
+
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build.VERSION_CODES;
+import android.security.KeyPairGeneratorSpec;
+import android.util.Base64;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.KeyStore;
+import java.security.KeyStore.Entry;
+import java.security.KeyStore.PrivateKeyEntry;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.PublicKey;
+import java.util.ArrayList;
+import java.util.Calendar;
+import javax.crypto.Cipher;
+import javax.crypto.CipherInputStream;
+import javax.crypto.CipherOutputStream;
+import javax.crypto.NoSuchPaddingException;
+import javax.security.auth.x500.X500Principal;
+
+class EncryptorImplV18 implements Encryptor {
+
+    private static final String ANDROID_KEY_STORE = "AndroidKeyStore";
+    private static final String RSA_MODE = "RSA/ECB/PKCS1Padding";
+    private static final String CIPHER_PROVIDER = "AndroidOpenSSL";
+    private static final String ALIAS = "KinKeyStoreRSA";
+    private static final String RSA = "RSA";
+
+    private final Context context;
+
+    EncryptorImplV18(Context context) {
+        this.context = context;
+    }
+
+    public String encrypt(String secret) throws CryptoException {
+        try {
+            KeyStore keyStore = KeyStore.getInstance(ANDROID_KEY_STORE);
+            keyStore.load(null);
```

Comment:
?

Reply:
that's a load method with load params that can be null, in our case, it should be null